checkbox-ng/checkbox_ng/launcher/stages.py

```diff
@@ -180,7 +181,7 @@ def _run_single_job_with_ui_loop(self, job, ui):
                                 result_builder.comments = comments
                             break
                     elif cmd == "quit":
-                        raise SystemExit()
+                        raise SystemExit("Session saved")
```

Comment:
```suggestion
                        raise SystemExit("Session paused by the operator")
```

Reply:
I wanted to make this uniform across local and remote, do you want me to replace the remote one as well?

Reply:
Yes. And maybe let's not use the word "operator", as it may be confusing to some (too close to "controller"). Let's use "user". I want to put emphasis on the fact that a human did it :)

Reply:
done